### PR TITLE
Correct typo in SPEC_PREFIX-NUMBER

### LINKED ISSUE #43: Special ED Active table not rendered
## Action Items
1. Identify why the Special ED Active table is not rendered for student number 523163.
    - ✅ https://github.com/tulsaschoolsdata/tps-legacy-ecumulative-reports/blame/67768795647d24374560bc41fa5e719a38f0a8fc/templates/macros.njk#L638
2. Identify how many student records are affected by the underlying issue.
3. Regenerate PDFs for affected student records only.
4. Upload only regenerated PDFs to Scribbles.

## References
- [Teams thread](https://teams.microsoft.com/l/message/19:meeting_MDNiNmM3YmYtYWJkZi00YzBlLTgzMjAtNWFmZjM4ZTJlZGE2@thread.v2/1693424094362?context=%7B%22contextType%22%3A%22chat%22%7D)
- https://github.com/search?q=repo%3Atulsaschoolsdata%2Ftps-legacy-ecumulative-reports%20spd_special_ed&type=code

### PR DESCRIPTION
**Pull request by jasonben**
closes #43

A file containing only sped students was generated which contains 1 student_number per line. This PR adds the ability to read student_numbers from stdin. New reports were generated and delivered to Justin A by @zuvy.

### CODE REVIEW

**Review by zuvy — COMMENTED** [review]
lgtm

**Review by travismiller — DISMISSED** [review]
A couple of things:
- ⛔️ Let's not commit the student list file into the repo
- 💡 The csv package probably isn't necessary for this use case

**Comment by travismiller** [comment]
For reading in the file, I‘ll recommend a read from stdin approach like this: https://stackoverflow.com/a/76743097/1596013

The interface would be like:

```console
$ npx ts-node src/script.ts < student_numbers.txt
```

File: src/script.ts
Comment (jasonben) [inline src/script.ts:1]: Adds the stdin feature

**Review by zuvy — APPROVED** [review]
Good to go.

**Pull request merged**